test/spec/sparkpost.spec.js

```diff
@@ -248,14 +255,41 @@ describe('SparkPost Library', function() {
         done();
       });
     });
+
+    it('should return a parsed JSON object', function(done) {
+      nock('https://test.sparkpost.com')
+        .get('/test')
+        .reply(200, '{ "ok": true }');
```

Comment:
Very minor: Do we need separate string and an object versions of the response object?  We could just keep it in a var and encode/decode/compare as necessary.